docs/guides/data-writes.md

```diff
@@ -292,7 +292,7 @@ You can ship this code as-is. The browser will handle the pending UI and interru
 
 ### Graduate to `<Form>` and add pending UI
 
-Let's use progressive enhancement to make this UX a bit more fancy. By changing it from `<Form reloadDocument>` to `<Form>`, Remix will emulate the browser behavior with `fetch`. It will also give you access to the pending form data so you can build pending UI.
+Let's use progressive enhancement to make this UX a bit more fancy. By changing it from `<form>` to `<Form>`, Remix will emulate the browser behavior with `fetch`. It will also give you access to the pending form data so you can build pending UI.
```

Comment:
I think the idea was to imply that `<form/>` is equivalent to `<Form reload document />`. Might be worth adding this in a parenthesis (or.. ).

WDYT @kentcdodds ?

Reply:
The issue here is that `<Form reloadDocument />` hasn't been mentioned even once, either in text nor code. So the reader following along with the guide will have `<form />` implemented in their editor at this point.

The paragraph below this one does mention the differences between `form` and `Form`, and the necessity of adding `reloadDocument` to get better UX. This mini-PR doesn't take a stance on that part :)